source/developers/form-sources/form-source-mediaconvert-transcode.rst

```diff
@@ -30,9 +30,10 @@ Configuration
 .. include:: /includes/form-sources/form-source-field-basics.rst
 
 +------------------------+-----------------------------------------------------------------------------+
-|| Description/Purpose   || Data source to upload videos to WebDav repository                          |
+|| Description/Purpose   || Data source to upload videos to AWS MediaConvert repository                |
 +------------------------+-----------------------------------------------------------------------------+
-|| Properties            || - Input Profile ID: AWS profile id (MediaConvert) to be used for storing   |
-||                       ||                     the new transcoded videos setup in aws.xml             |
-||                       || - Output Profile ID: S3 profile id to be used setup in aws.xml             |
+|| Properties            || - Input Profile ID: AWS profile id (MediaConvert) to be used to access the |
```

Comment:
it's the other way around, `input` is used for uploading & triggering and `ouput` for the generated files